neqo-transport/src/fc.rs

```diff
@@ -255,6 +273,11 @@ where
         }
     }
 
+    const fn should_send_flowc_update(&self) -> bool {
+        let window_bytes_unused = self.max_allowed.saturating_sub(self.retired);
```

Comment:
Is it ever the case that `max_allowed < retired` such that you need `saturating_sub()`?

Reply:
Fixed in 90a7a2039983979f81d37b84befe376add19d01c. Sorry for the workaround instead of a fix in the first place.